### PR TITLE
Showraise gtk gtk3

### PR DESCRIPTION
**Pull request by fariza**
Following https://github.com/matplotlib/matplotlib/pull/6384 in response to https://github.com/matplotlib/matplotlib/issues/6378

The gtk trick is a little bit more obscure because not all methods work in all versions of pygtk, I hope this one will work for everyone

### CODE REVIEW

**Comment by fariza** [comment]
@efiring I did a separate PR, is this OK?

**Comment by efiring** [comment]
Yes, a separate PR is fine.  Thanks for doing this.

File: lib/matplotlib/backends/backend_gtk.py [hunk 1]
```diff
@@ -613,6 +613,8 @@ def destroy(self, *args):
     def show(self):
         # show the figure window
         self.window.show()
+        self.window.set_keep_above(True)
```
Comment (jenshnielsen) [inline lib/matplotlib/backends/backend_gtk.py]: It's probably useful to document with a comment that this is intended so that no one "optimises" it away

**Comment by jenshnielsen** [comment]
Appveyor not relevant fixed by #6423

**Pull request merged**